File: beecrowd/c/bee1113.c
#include <stdio.h>

int main() {

    int x, y;

    for (;;) {
        scanf("%d %d", &x, &y);

        if (x == y) break;

        printf("%s\n", (x < y ? "Crescente" : "Decrescente"));
    }

    return 0;
}

File: beecrowd/go/bee1113.go
package main

import "fmt"

func main() {

	var x, y int

	for {
		fmt.Scanf("%d %d", &x, &y)

		if x == y {
			break
		}

		if x < y {
			fmt.Println("Crescente")
		} else {
			fmt.Println("Decrescente")
		}
	}

}
